decidim-core/app/uploaders/decidim/avatar_uploader.rb

```diff
@@ -7,11 +7,15 @@ class AvatarUploader < ImageUploader
 
     process :validate_dimensions
 
-    version :big do
+    version :profile do
+      process resize_to_fill: [268, 320]
```

Comment:
Sorry about this, but we should probably be resizing to double this size (536x640) for retina displays :rage3: